projects/core/src/lib/time/time-step-selector/time-step-selector.component.scss

```
@@ -6,6 +6,11 @@
     }
 }
 
+.padded {
+    padding: 1rem;
+    padding-left: 2rem;
```

```suggestion
    padding: 1rem 1rem 1rem 2rem;
```

Do you know why you have to adjust it more to the left?

---

That's because the `.container` span property has a right margin of 1rem, without which the box would end directly after the timestamp. Is it okay this way, or should I try to reorganize the values so that this right/left offset is unified?